wp-content/themes/core/blocks/tribe/copyright/style.pcss

```diff
@@ -0,0 +1,14 @@
+/**
+ * The following styles get applied both on the front of your site
+ * and in the editor.
+ *
+ * Replace them with your own styles or remove the file completely.
+ */
+
+/* -----------------------------------------------------------------------
+ * Copyright Block
+ * ----------------------------------------------------------------------- */
+
+p.wp-block-tribe-copyright {
+	margin-top: 0;
+}
```

Comment:
❓ If we're including this block by default can we set this in `theme.json` instead? One less file to keep track of.